components/infobox/wikis/apexlegends/infobox_weapon_custom.lua

```diff
@@ -72,6 +81,25 @@ function CustomInjector:addCustomCells(widgets)
 		content = {args.rateoffireauto}
 	})
 
+	table.insert(widgets, Cell{
+		name = 'Rate of fire (Burst)',
+		content = {args.rateoffireburst}
+	})
+
+	if String.isNotEmpty(args.ratesoffire) and String.isEmpty(args.ratesoffireauto) then
```

Comment:
Might want to check `rateoffire1` also, instead of just rateoffire

Reply:
it checks "ratesoffire" with an "s". And from input given from Fenrir in discord, his advice was naming conventions should start as blank, then 2, 3, etc

Reply:
Since `_weapon:getAllArgsForBase` handles both, only changing the if statement slightly is required, but I'll leave it up to you

Reply:
I think then I'll leave it to be consistent with the naming convention of the other parameters, if I'm understanding correctly.
Also, the most recent edit of line 89 was a change not related to this conversation.